src/components/modules/history/hooks/useHistoryQuery.ts

```
@@ -0,0 +1,97 @@
+      return "Escrow funds released";
+    case "resolved":
+      return "Dispute resolved";
+    case "disputed":
+      return "Dispute started";
+    case "milestone_completed":
+      return "Milestone completed";
+    case "milestone_approved":
+      return "Milestone approved";
+    case "funded":
+      return "Escrow funded";
+    case "created":
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)